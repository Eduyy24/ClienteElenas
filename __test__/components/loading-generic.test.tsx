import React from 'react';
import { render } from '@testing-library/react-native'
import LoadingGeneric from '../../src/components/loading-generic';

describe('<LoadingGeneric />', () => {
  test('Ui del componente', ()=> {
    const wraper = render(<LoadingGeneric />)
    expect(wraper.toJSON()).toMatchSnapshot()
  })
})