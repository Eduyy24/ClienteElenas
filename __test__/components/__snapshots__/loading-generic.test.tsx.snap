// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingGeneric /> Ui del componente 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 16,
        "fontWeight": "bold",
      }
    }
  >
    Cargando...
  </Text>
</View>
`;
